demos/select.html

```diff
@@ -93,56 +93,98 @@
         </div>
       </section>
 
-      <section class="example">
-        <h2 class="mdc-typography--title">Fully-Featured Component</h2>
-        <section id="box-demo-wrapper">
-          <div id="js-select-box" class="mdc-select" role="listbox">
-            <div class="mdc-select__surface" tabindex="0">
-              <div class="mdc-select__label">Food Group</div>
-              <div class="mdc-select__selected-text"></div>
-              <div class="mdc-select__bottom-line"></div>
+        <section class="example">
```

Comment:
Every block here has an extra indent (I think you need to set your tabs to be 2 spaces instead of 4.)